src/SixLabors.Fonts/TextRenderer.cs

```diff
@@ -59,11 +59,6 @@ public void RenderText(ReadOnlySpan<char> text, TextOptions options)
 
             foreach (GlyphLayout g in glyphsToRender)
             {
-                if (g.IsWhiteSpace())
```

Comment:
we have to render all glyphs even whitespace to handle underline/strikeout for the spaces, I now skip rendering the glyph outlines in the glyph `RenderTo` methods instead.